src/renderer/core/canvas/links/linkConnectorAdapter.ts

```diff
@@ -72,11 +73,20 @@ export class LinkConnectorAdapter {
   beginFromInput(
     nodeId: NodeId,
     inputIndex: number,
-    opts?: { moveExisting?: boolean; fromRerouteId?: RerouteId }
+    opts?: {
+      moveExisting?: boolean
+      fromRerouteId?: RerouteId
+      layout: SlotLayout
+    }
```

Comment:
Nit: Required before Optional
```suggestion
    opts?: {
      layout: SlotLayout
      moveExisting?: boolean
      fromRerouteId?: RerouteId
    }
```